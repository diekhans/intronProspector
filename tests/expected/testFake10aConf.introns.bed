fake10	487560	488264	sj8_GT/AG	37	+	487560	488264	0,128,0
fake10	488522	488947	sj9_GT/AG	46	+	488522	488947	0,128,0
fake10	489118	490086	sj1_GT/AG	44	+	489118	490086	0,128,0
