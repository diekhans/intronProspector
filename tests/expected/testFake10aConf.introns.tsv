chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites	name
fake10	487560	488264	+	37	0	0	68	90	3.92	GT/AG	sj8
fake10	488522	488947	+	44	2	0	87	91	4.47	GT/AG	sj9
fake10	489118	490086	+	42	2	0	93	90	4.85	GT/AG	sj1
